main.c

```diff
@@ -151,6 +151,10 @@ void check_help(char **arg)
   }
 }
 
+#ifdef __NetBSD__
```

Comment:
:broom: NIT - Consider a consistent macro style

Similar to other files; TL;DR

Please consider:

```suggestion
#if defined(__NetBSD__) && !defined(uselocale)
// use a No-OP for NetBSD instead of uselocale() for compatability 
```